### PR TITLE
moving OpenIDUserRPDAO to identity core and packing slf4j in the bundles that pack oltu

### PR DESCRIPTION
**Pull request by madurangasiriwardena**

### CODE REVIEW

File: components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDUserRPDAO.java [hunk 1]
```diff
@@ -348,50 +299,99 @@ public OpenIDUserRPDO getOpenIDUserRP(String userName, String rpUrl) throws Iden
...
-                if (log.isDebugEnabled()) {
-                    log.error("Error while parsing resourceToObject", e);
-                }
+    private OpenIDUserRPDO buildUserRPDO(ResultSet results, String userName) {
+
+        OpenIDUserRPDO rpdo = new OpenIDUserRPDO();
+
+        try {
+            if (!results.next()) {
+                log.debug("RememberMe token not found for the user " + userName);
```
Comment (omindu) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDUserRPDAO.java]: Check ```isDebugEnabled()```

Comment (madurangasiriwardena) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDUserRPDAO.java]: fixed with https://github.com/wso2/carbon-identity/pull/1679

File: components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDSQLQueries.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ *  under the License.
+ * /
+ */
+
+package org.wso2.carbon.identity.core.dao;
+
+/**
+ * SQL Queries required for the AssociationDAO.
+ *
+ * @author WSO2 Inc.
```
Comment (Pushpalanka) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDSQLQueries.java]: No need to have author tag

Comment (madurangasiriwardena) [inline components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/OpenIDSQLQueries.java]: Fixed with https://github.com/wso2/carbon-identity/pull/1679

**Pull request merged**